<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>450</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>431</height>
    </rect>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>COM Ports</string>
    </attribute>
    <widget class="QGroupBox" name="fanControlGroupBox">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>271</width>
       <height>381</height>
      </rect>
     </property>
     <property name="title">
      <string>Fan Control Port</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>191</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Baud Rate:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Data Bits:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Parity:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="fanParityComboBox">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Even</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Odd</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Space</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mark</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Flow Control:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="fanFlowComboBox">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hardware Control</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Software Control</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Stop Bits:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="fanStopBitsComboBox">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="fanPortComboBox">
         <property name="maxVisibleItems">
          <number>20</number>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="fanBaudComboBox">
         <item>
          <property name="text">
           <string>1200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="fanDataBitsComboBox">
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="powerSupplyGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>381</height>
      </rect>
     </property>
     <property name="title">
      <string>Power Supply Port</string>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>183</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Baud Rate:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Data Bits:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Parity:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="supplyParityComboBox">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Even</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Odd</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Space</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mark</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Flow Control:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="supplyFlowComboBox">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hardware Control</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Software Control</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Stop Bits:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="supplyStopBitsComboBox">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="supplyPortComboBox">
         <property name="maxVisibleItems">
          <number>20</number>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="supplyBaudComboBox">
         <item>
          <property name="text">
           <string>1200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="supplyDataBitsComboBox">
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
